github/ci/testgrid/gen-config.yaml

```diff
@@ -1,11 +1,13 @@
 dashboards:
+- name: kubevirt-containerized-data-importer
```

Comment:
So you don't want to split dashboards for cdi as we do with kubevirt?

Reply:
if you mean separate between periodics and presubmits - cdi doesn't seem to have periodics

Reply:
Not right now, but we might in the future, I just added a post submit yesterday, and was considering a daily periodic while I did that. So it makes sense to prepare for that.